core/lib/makeswift/components/customer-group-slot/customer-group-slot.makeswift.ts

```diff
@@ -0,0 +1,93 @@
+import { Combobox, Group, List, Slot, Style } from '@makeswift/runtime/controls';
```

Comment:
Here and below, I'm simply splitting up `<component>.makeswift.tsx` into  `<component>.makeswift.ts` and  `<component>.tsx`, but for some reason GitHub doesn't recognize the rename and shows it as deletion + addition.

There are no other semantic changes.